pyServer/AzureDiskInspectService.py

```diff
@@ -663,7 +680,10 @@ def do_POST(self):
             failureStatusText = 'Server Error'
         finally:
             if (not requestSucceeded):
-                self.send_error(failureResultCode, "%s: %s" % (failureStatusText, str(telemetryException)))
+                # Change error response default format from html to json
+                self.error_message_format = json.dumps(ErrorDetails("%(explain)s", "%(message)s").__dict__)
```

Comment:
Maybe add a link to the reference doc on how the error message formatter works.

Reply:
Added reference doc link.